app/app/src/main/res/layout/activity_main.xml

```diff
@@ -11,6 +11,11 @@
 
     <include layout="@layout/content_eeg" />
 
-    <include layout="@layout/content_joystick" />
+    <include
+        layout="@layout/content_joystick"
+        android:layout_width="match_parent"
+        android:layout_height="40dp" />
```

Comment:
This is already stated within the included layout. Stating height twice creates a redundancy.